### PR TITLE
[PL] Added support for more entity in weather (detailed weather, temperature, humidity, dew point, pressure, wind speed, ozone, cloud coverage, visibility, uv index)

### PR DESCRIPTION
**Pull request by witold-gren**
Added by default support for:
- detailed weather
- temperature
- humidity
- dew point
- pressure
- wind speed
- ozone
- cloud coverage
- visibility
- uv index

### CODE REVIEW

**Comment by Uriziel01** [comment]
@witold-gren I'll have a look at it when back at home, today's night/tomorrow's morning 👍

File: sentences/pl/weather_HassGetWeather.yaml [hunk 1]
```diff
@@ -3,8 +3,95 @@ intents:
...
+          - "Ile [teraz ]wynosi [aktualna | obecna ]wilgotność powietrza"
+          - "Jaki jest [aktualny | obecny ]poziom wilgotności[ powietrza]"
+          - "Jaka jest [aktualna | obecna ]wilgotność[ powietrza]"
+          - "[Podaj mi ][aktualną | obecną ]wilgotność powietrza[ na zewnątrz| na dworze| na polu| na placu]"
+        response: humidity
+
+      - sentences:
+          - "Jaki jest [obecny |aktualny ]poziom punktu rosy"
+          - "Jaki jest [obecny |aktualny ]punkt rosy"
+          - "Sprawdzić punkt rosy[ na zewnątrz| na dworze| na polu| na placu]"
```
[the remark this answers is not in the document]
Comment (witold-gren) [reply sentences/pl/weather_HassGetWeather.yaml]: good point, I added this words to sentences related with dew point and temperature

File: responses/pl/HassClimateGetTemperature.yaml [hunk 2]
```diff
@@ -4,18 +4,69 @@ responses:
     HassClimateGetTemperature:
       default: >
         {% set temperature = state_attr(state.entity_id, 'current_temperature') %}
-        {% set temperature_pl = (temperature|string).replace(".", ",") %}
-        Temperatura wynosi {% if temperature_pl.startswith("-") %}minus {% endif %}{{ temperature_pl.replace("-", "") }} {{ 'stopień' if temperature | int | abs == 1 else 'stopnie' if temperature | int | abs in [2,3,4] else 'stopnie' if (temperature|string)[-1] | int | abs in [2,3,4] else 'stopni' }}
+        {% set temperature_pl = (temperature | string).replace(".", ",") %}
+        {% set temperature_decmal = (temperature | float - temperature | int) | round(2) %}
```
[the remark this answers is not in the document]
Comment (witold-gren) [reply responses/pl/HassClimateGetTemperature.yaml]: yep, updated

File: responses/pl/HassClimateGetTemperature.yaml [hunk 3]
```diff
@@ -4,18 +4,69 @@ responses:
...
+            {% set degree_name = 'stopnie' %}
+          {% elif (temperature | int | string)[-1] | int | abs in [2, 3, 4] %}
+            {% set degree_name = 'stopnie' %}
+          {% else %}
+            {% set degree_name = 'stopni' %}
+          {% endif %}
+        {% else %}
+          {% set degree_name = "stopnia" %}
+        {% endif %}
+        Temperatura wynosi {{ negative_name }}{{ temperature_pl.replace("-", "") }} {{ degree_name }}
```
[the remark this answers is not in the document]
Comment (witold-gren) [reply responses/pl/HassClimateGetTemperature.yaml:28]: I added extra statement for properly support value 12,13,14. I also updated other places in weather to fix this issue everywhere where we ask about temperature

**Review by Uriziel01 — APPROVED** [review]

**Comment by Uriziel01** [comment]
@witold-gren Approved, merge at your discretion 🙂

**Pull request merged**